perfmetrics/scripts/fetch_metrics.py

```diff
@@ -94,5 +122,10 @@ def _parse_arguments(argv):
     for row in metrics_data:
       vm_metrics_data.append(row)
 
-  if args.upload:
-    gsheet.write_to_google_sheet(VM_WORKSHEET_NAME, vm_metrics_data)
+  vm_metrics_data_upload = [row[1:] + [None, None, None, None, None, None, None, None] for row in vm_metrics_data]
+  if args.upload_gs:
+    gsheet.write_to_google_sheet(VM_WORKSHEET_NAME, vm_metrics_data_upload)
+  if args.upload_bq:
+    if not args.config_id or args.start_time_build:
+      raise Exception("Pass required arguments experiments configuration ID and start time of build for uploading to BigQuery")
+    bigquery_obj.upload_metrics_to_table('vm', args.config_id[0], args.start_time_build[0], vm_metrics_data_upload)
```

Comment:
Do you also think a better way could be :
1. Define the table ids in bigquery/constants.py as constants.
2. Import the constants here and wherever required.
3. Remove the mapping from table name to table id in Bigquery class.

Reply:
That would remove some magic constants. We can do this! But then should we also remove the class constants from ExperimentsGCSFuseBQ class?